src/public/services/FormService.ts

```diff
@@ -0,0 +1,134 @@
+import axios from 'axios'
+
+import { FormUpdateParams } from 'src/app/modules/form/admin-form/admin-form.types'
+import { PublicFormViewDto } from 'src/app/modules/form/public-form/public-form.types'
+import {
+  FormMetaView,
+  IFormSchema,
+  IPopulatedForm,
+  PublicForm,
+} from 'src/types'
+import { FormFieldDto } from 'src/types/api'
+
+// endpoints exported for testing
+export const ADMIN_FORM_ENDPOINT = '/api/v3/admin/forms'
+export const PUBLIC_FORM_ENDPOINT = '/api/v3/forms'
+
+export const queryForm = async (): Promise<FormMetaView[]> => {
+  const instance = axios.create()
+  return instance
+    .get<FormMetaView[]>(`${ADMIN_FORM_ENDPOINT}`, {
+      headers: { 'If-Modified-Since': '0' },
+    })
+    .then(({ data }) => data)
+}
+
+export const getAdminForm = async (
+  formId: string,
+  accessMode = 'adminform',
+): Promise<{ form: IPopulatedForm }> => {
+  const instance = axios.create()
+  // disable IE ajax request caching (so new forms show on dashboard)
+  return instance
+    .get<{ form: IPopulatedForm }>(`/${formId}/${accessMode}`, {
+      headers: { 'If-Modified-Since': '0' },
+    })
+    .then(({ data }) => data)
+}
+
+export const getPublicForm = async (
+  formId: string,
+): Promise<PublicFormViewDto> => {
+  const instance = axios.create()
+  // disable IE ajax request caching (so new forms show on dashboard)
+  return instance
+    .get<PublicFormViewDto>(`${PUBLIC_FORM_ENDPOINT}/${formId}`, {
+      headers: { 'If-Modified-Since': '0' },
+    })
+    .then(({ data }) => data)
+}
+
+export const updateForm = async (
+  formId: string,
+  update: { form: FormUpdateParams },
+  accessMode = 'adminform',
+): Promise<IPopulatedForm> => {
+  const instance = axios.create()
+  return instance
+    .put<IPopulatedForm>(`${formId}/${accessMode}`, update)
+    .then(({ data }) => data)
+}
+
+export const saveForm = async (
+  formId: string,
+  formToSave: FormFieldDto[],
+): Promise<FormMetaView> => {
+  const instance = axios.create()
+  return instance
+    .post<FormMetaView>(
+      `${ADMIN_FORM_ENDPOINT}/${formId}/duplicate`,
+      formToSave,
+    )
+    .then(({ data }) => data)
+}
+
+export const deleteForm = async (formId: string): Promise<void> => {
+  const instance = axios.create()
+  return instance.delete(`${ADMIN_FORM_ENDPOINT}/${formId}`)
+}
+
+// createForm is called without formId, so the endpoint is just /adminform
+export const createForm = async (newForm: {
+  form: FormFieldDto[]
+}): Promise<IFormSchema> => {
+  const instance = axios.create()
+  return instance
+    .post<IFormSchema>(`${ADMIN_FORM_ENDPOINT}`, newForm)
+    .then(({ data }) => data)
+}
+
+// Used for viewing templates with use-template or examples listing. Any logged in officer is authorized.
+export const queryTemplate = async (
```

Comment:
in the jsdocs, could you flag which service functions are used purely by the examples page? this is to make it easier for us to know which functions can be deleted in React, as we don't plan to maintain the Examples page in v3